php_1/db/PdoConnectionClass.php

```diff
@@ -1,16 +1,25 @@
 <?php
+require_once 'Config.php';
+
 class PdoConnectionClass
 {
-    private $server = "mysql:host=localhost;dbname=mydatabase3";
-    private $user = "root";
-    private $pass = "";
+    private $serverName = Config::HOST;
+    private $database = Config::DATABASE;
+    private $user = Config::USER;
+    private $pass = Config::PASSWORD;
     private $options = array(PDO::ATTR_ERRMODE => PDO::ERRMODE_EXCEPTION, PDO::ATTR_DEFAULT_FETCH_MODE => PDO::FETCH_ASSOC);
     protected $con;
+    protected $db;
```

Comment:
do not need this variable. We can use $this->con which is set in the connect() function.